test/start.test.js

```diff
@@ -9,6 +9,7 @@ const semver = require('semver')
 const baseFilename = path.join(__dirname, 'fixtures', `test_${crypto.randomBytes(16).toString('hex')}`)
 const { fork } = require('child_process')
 const moduleSupport = semver.satisfies(process.version, '>= 14 || >= 12.17.0 < 13.0.0')
+require('events').EventEmitter.defaultMaxListeners = 0
```

Comment:
can you add comment to this line because it's not self-explanatory why it's there?

Reply:
I was running into these warnings when I was testing the implementation. `MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 SIGTERM listeners added to [process]. Use emitter.setMaxListeners() to increase limit`

I tried with: afterEachTest

Reply:
If we are actually leaking listeners somewhere, that's rather bad, we are likely to cause warnings for user tests downstream then.

Reply:
Unfortunately this means that we are not cleaning things up properly. You might need to call `uninstall` - see https://github.com/mcollina/close-with-grace#return-values.

Reply:
Thanks for the feedback, I've included the `uninstall` method inside of `onClose` hook and we don't get any memory leak warnings